app/src/main/java/com/app/calculator/presenter/OperationPresenter.java

```diff
@@ -27,7 +27,10 @@ private void onButtonPressed() {
         else model.setSecondNumber(Integer.parseInt("0"));
 
         if(!view.getOperator().equalsIgnoreCase(Operator.DEF.getSymbol())) {
```

Comment:
view.getOperator could just return the ENUM value, and the just `.equals(Operator.DEF)`